pkg/awsutils/awsutils.go

```diff
@@ -353,19 +326,8 @@ func New(useCustomNetworking bool) (*EC2InstanceMetadataCache, error) {
 	cache.useCustomNetworking = useCustomNetworking
 	log.Infof("Custom networking %v", cache.useCustomNetworking)
 
-	sess, err := session.NewSession(
-		&aws.Config{
-			Region: aws.String(cache.region), 
-			MaxRetries: aws.Int(15),
```

Comment:
Max retry was 15 here and 10 above. Do we know why we used different values for those two sessions?

Reply:
I am not sure why this discrepancy. I went ahead and assumed that 10 retries should be a good number to meet most of the use-cases. Let me know if you want me to increase it.